src/Workspaces/Core/Portable/CodeGeneration/CodeGenerationSymbolFactory.cs

```diff
@@ -423,6 +423,7 @@ public static INamespaceSymbol CreateNamespaceSymbol(string name, IList<ISymbol>
             DeclarationModifiers? modifiers = null,
             ImmutableArray<IMethodSymbol> explicitInterfaceImplementations = default,
             string name = null,
+            ImmutableArray<IParameterSymbol>? parameters = default,
```

Comment:
nit:
`ImmutableArray<IParameterSymbol>? parameters = null` just to be consistent with others

Reply:
ImmutableArray is a struct, so we need to use `default` which is also above for `explicitInterfaceImplementations`

Reply:
This is a `Nullable<T>` though :)

Reply:
Touche!  Why is it nullable though?  an ImmutbaleArray can already represent '3' states (null/emty/elements), we do we need an additional Nullable on top of that?

Reply:
1. ImmutableArray is a struct: default should be used not null.
2. ImmutableArray has enough states to distinguish cases we consider: default, empty, items.

No changes required. Thank you!

---
In reply to: [290923010](https://github.com/dotnet/roslyn/pull/36163#discussion_r290923010) [](ancestors = 290923010)